src/cli/commands/lint.ts

```diff
@@ -100,26 +109,38 @@ linting ./openapi.yaml
   }
 }
 
-async function lint(name: string, flags: any, command: Lint, customRules?: RuleCollection) {
+async function lint(name: string, flags: any, command: Lint, rules?: RuleCollection) {
   command.log(`Linting ${name}`);
   const spec: IParserResult = await readParsable(name, flags.encoding);
 
   const spectral = new Spectral();
-  if (customRules) {
-    command.log('Applying custom rules. Automatic rule detection is off.');
-    spectral.addRules(customRules);
-  } else if (parseInt(spec.data.swagger) === 2) {
-    command.log('OpenAPI 2.0 (Swagger) detected');
-    spectral.addFunctions(oas2Functions());
-    spectral.addRules(oas2Rules());
-  } else if (parseInt(spec.data.openapi) === 3) {
-    command.log('OpenAPI 3.x detected');
-    spectral.addFunctions(oas3Functions());
-    spectral.addRules(oas3Rules());
+  if (rules !== undefined) {
+    const numRules = new Array(rules).length;
+    command.log(`Found ${numRules} rules`);
```

Comment:
I'd say it should be only printed when `verbose` is enabled.
Doesn't feel like it's very important info.

Reply:
Done!